core/src/main/java/com/github/mizool/core/password/Pbkdf2WithHmacSha1Hasher.java

```diff
@@ -67,12 +68,11 @@ public String toString()
 
     public static final String ALGORITHM_NAME = "PBKDF2WithHmacSHA1";
 
-    @VisibleForTesting
-    protected static final String ITERATIONS_PROPERTY_NAME = Pbkdf2WithHmacSha1Hasher.class.getName() + ".iterations";
+    private static final PropertyNode CONFIG = Config.systemProperties()
```

Comment:
Inline this, as it's only used once. The old extracted property name had the purpose to reduce nested calls. The new design chains calls, which I don't see as a problem.